### DISCUSSION TITLE
How to setup Suricata

### DISCUSSION THREAD

**Discussion by k-alki** [Security-Onion-Solutions/securityonion#11378] [2.4]
### Version

2.4.10

### Installation Method

Network installation on Ubuntu

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

16

### Storage for /

250Go

### Storage for /nsm

50

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello all !

I am beginning with onion security and i'm struggling for integrating the suricata logs. Is there any walkthrough somewhere ?

Thank you !!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
If you installed in Standalone mode, then Suricata should already be configured and sniffing whatever NIC you chose as your monitoring interface. Those Suricata logs should then be automatically ingested into Elasticsearch and you should be able to see them by logging into Security Onion Console. For more information, please see:
https://docs.securityonion.net/en/2.4/hardware.html#standalone-deployments
https://docs.securityonion.net/en/2.4/architecture.html#standalone
https://docs.securityonion.net/en/2.4/configuration.html#production-server-standalone
https://docs.securityonion.net/en/2.4/elasticsearch.html
https://docs.securityonion.net/en/2.4/soc.html
